src/runtime_src/driver/xclng/drm/xocl/mgmtpf/mgmt-core.c

```diff
@@ -362,7 +362,7 @@ static struct xclmgmt_char *create_char(struct xclmgmt_dev *lro)
 	/* couple the control device file operations to the character device */
 	cdev_init(&lro_char->cdev, &ctrl_fops);
 	lro_char->cdev.owner = THIS_MODULE;
-	lro_char->cdev.dev = MKDEV(MAJOR(xclmgmt_devnode), lro->instance);
+	lro_char->cdev.dev = MKDEV(MAJOR(xclmgmt_devnode), lro->core.dev_minor);
```

Comment:
Can we just keep one "id", either "instance" or "core.dev_minor"? They seem to serve the same purpose.

Reply:
The maximum minor is 20bits. However, pci id is 32bits which we used as device instance name. e.g. /dev/xclmgmt<instance>. So I would still use pci id as char dev name/sysfs name, but another minor id with is invisible for end user.